Attributes/ValidationAttributes/MaxFileSizeAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUkDesignSystem.Attributes.ValidationAttributes
+{
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    public class MaxFileSizeAttribute : ValidationAttribute
+    {
+        private readonly int _maxFileSize;
+        public MaxFileSizeAttribute(int maxFileSize)
+        {
+            _maxFileSize = maxFileSize;
+        }
+
+        public override bool IsValid(object value)
```

Comment:
We want to override the version that returns a ValidationResult so that we can set the error message text ourselves